modules/nf-core/seqfu/derep/main.nf

```diff
@@ -0,0 +1,46 @@
+process SEQFU_DEREP {
+    tag "$meta.id"
+    label 'process_low'
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/seqfu:1.20.3--h1eb128b_2':
+        'biocontainers/seqfu:1.20.3--h1eb128b_2' }"
+
+    input:
+    tuple val(meta), path(fastas)
+
+    output:
+    tuple val(meta), path("*_derep.fasta.gz"), emit: fasta
+    path "versions.yml"                      , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+
+    """
+    seqfu \\
+        derep \\
+        $args \\
+        $fastas | gzip -c > "${meta.id}_derep.fasta.gz"
+
+    cat <<-END_VERSIONS > versions.yml
+    "${task.process}":
+        seqfu: \$(seqfu version)
+    END_VERSIONS
+    """
+
+    stub:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    echo "" | gzip -c > "${prefix}_derep.fasta.gz"
```

Comment:
Thanks @SPPearce !
Can you elaborate a bit on this? It could be useful to expand that section in https://nf-co.re/docs/guidelines/components/modules Cheers